packages/berlin/src/components/tables/groups-table/GroupsTable.tsx

```diff
@@ -79,9 +86,39 @@ function GroupCard({ userToGroup, theme, onLeaveGroup }: GroupCardProps) {
       />
       <FlexColumn className="description" $gap="1.5rem">
         <Body>
-          <Bold>Group members:</Bold> {groupMembers?.map((member) => member.username)}
+          <Bold>Group members:</Bold> {groupMembers?.map((member) => member.username).join(', ')}
         </Body>
-        <Body>More details here...</Body>
+        {groupRegistrations &&
+          groupRegistrations.flatMap((groupRegistration) =>
+            groupRegistration.registrations.flatMap((registration) =>
+              registration.registrationData
+                .filter(
+                  ({ registrationField }) =>
+                    registrationField.fieldDisplayRank === 0 ||
+                    registrationField.fieldDisplayRank === 1,
+                )
+                .map(({ id, value, registrationField }) => {
+                  if (registrationField.fieldDisplayRank === 0) {
+                    return (
+                      <div key={id}>
+                        <Body>
+                          <Bold>Title:</Bold> {value}
+                        </Body>
+                      </div>
+                    );
+                  } else if (registrationField.fieldDisplayRank === 1) {
+                    return (
+                      <div key={id}>
+                        <GroupProposalDescription>
+                          <Bold>Description:</Bold> {value}
+                        </GroupProposalDescription>
+                      </div>
+                    );
+                  }
```

Comment:
you could say something like:
`registrationField.name`: {value}

and have no if case